ReadMe.md

```diff
@@ -146,6 +146,46 @@ We are primarily considering JavaScript (sorry, not TypeScript) as the base lang
 * Aditya Pahilwani ([LinkedIn](https://www.linkedin.com/in/aditya-pahilwani-93b42bb5/))
 * Tejan Singh ([LinkedIn](https://www.linkedin.com/in/tejansingh/))
 
+## Contributor Code of Conduct
+
+### Our Pledge
```

Comment:
Can we have this in a separate file and link it from here? :) Thanks.

Reply:
I actually didn't get what you are suggesting 😅 Are you suggesting that I should create a separate ReadMe file?

Reply:
Separate Code of Conduct file. 😊  See the Contribution Guidelines section... 🔥

Reply:
okay!! I will surely follow that.

Reply:
Please check files changed section and any changes if required. 😀

Reply:
Will check now.